barretenberg/cpp/src/barretenberg/vm2/tracegen/lib/interaction_builder.hpp

```diff
@@ -10,4 +10,14 @@ class InteractionBuilderInterface {
     virtual void process(TraceContainer& trace) = 0;
 };
 
+// We set a dummy value in the inverse column so that the size of the column is right.
+// The correct value will be set by the prover.
+template <typename LookupSettings> void SetDummyInverses(TraceContainer& trace)
```

Comment:
I factored this out. It's clearer and it shouldn't hurt the theoretical complexity.